assets/js/modules/adsense/components/dashboard/DashboardTopEarningPagesWidgetGA4.js

```diff
@@ -295,7 +300,11 @@ function DashboardTopEarningPagesWidgetGA4( {
 	];
 
 	return (
-		<Widget noPadding Footer={ Footer } ref={ updateTrackingRef }>
+		<Widget
+			noPadding
+			Footer={ ! viewOnlyDashboard ? Footer : undefined }
```

Comment:
This is actually unnecessary, as the `SourceLink` used in `Footer` will return `null` on view-only dashboard. It results in an empty footer which is consistent look-wise with other widgets. See `DashboardPopularKeywordsWidget` for example.

Reply:
Ah, indeed it does… I noticed that, but then saw there was a needless empty footer section when using the `Footer` component in view-only mode...

But, that's consistent with other widgets: https://google.github.io/site-kit-wp/storybook/develop/?path=/story/modules-searchconsole-widgets-searchfunnelwidgetga4--view-only-search-console-only-ready

So I guess it's a bigger change and I won't do it here.

Reply:
Weirdly now I can't even reproduce that! 😓

Alright, I've made the change 👍🏻